client/core/trade.go

```diff
@@ -2867,10 +2873,24 @@ func (t *trackedTrade) confirmRedemption(match *matchTracker) {
 
 	match.confirmRedemptionNumTries++
 
+	// Try not to use (*Core).feeSuggestion here, since it can incur an RPC
+	// request to the server. t.redeemFeeSuggestion is updated every tick and
+	// uses a rate directly from our wallet, if available. Only go looking for
+	// one if we don't have one cached.
+	var feeSuggestion uint64
+	if _, is := t.accountRedeemer(); is {
+		feeSuggestion = t.metaData.RedeemMaxFeeRate
+	} else {
+		feeSuggestion = t.redeemFeeSuggestion.get()
+	}
+	if feeSuggestion == 0 {
+		feeSuggestion = t.dc.bestBookFeeSuggestion(t.wallets.toWallet.AssetID)
+	}
```

Comment:
seems like copy-paste ? maybe wrap it in helper **func** like `trackedTrade. redeemFeeSuggestion()` ?

Reply:
I is. Sure. I think used in 3 places so good idea.

Reply:
or 2... added redemptionFee method https://github.com/decred/dcrdex/compare/d9084f18b7795eddd92fa5afecd0c82b111d9d1a..b946db78fa3409c28bd67c3b2df81498f5ed8e9e